Triangulation_on_sphere_2/include/CGAL/Delaunay_triangulation_on_sphere_2.h

```diff
@@ -1051,7 +1051,7 @@ Delaunay_triangulation_on_sphere_2<Gt, Tds>::
 circumcenter_on_sphere(const Face_handle f) const
 {
   CGAL_precondition(dimension() == 2);
-  CGAL_precondition(!is_ghost(f));
+//  CGAL_precondition(!is_ghost(f));
```

Comment:
You shouldn't leave comments like these in the code

Reply:
Ok, sorry

Reply:
But I will put the commit message as a comment in the code, otherwise it does seem as you saw it.

Reply:
> That is on purpose, and the commit message explains why...

That is the second time you put the explanation message in the commit message (see the first one: https://github.com/CGAL/cgal/pull/5659#discussion_r626716681). I think that is not the best practice to help review the code by others developers:
  - the code may be shipped using a tarball,
  - not all developers knows how to use `git-blame`,
  - and anyway, if the code needs explanation, then the comments should be in the code itself, in my opinion.